style.css

```diff
@@ -611,3 +620,808 @@ body {
   list-style: none;
   padding-left: 80px;
 }
+
+@media screen and (min-width: 768px) {
+  * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  ::-webkit-scrollbar-bottom {
+    display: none;
+  }
```

Comment:
- [ ] As stated in the project requirement and suggested by the previous reviewer You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Project design has still the same issues : 
![Capture](https://user-images.githubusercontent.com/93598189/193090693-729ba17c-1a3f-4a6e-83b6-d3b7ec67c688.PNG)

- [ ] You can also refer the last reviewer Loom video(https://www.loom.com/share/0b8f8edf20794ad68fb931f1ca3a24c0)